syntax = "proto3";
import "google/protobuf/empty.proto";
package lidar;

message PointI3 {
    float x=1;
    float y=2;
    float z=3;
    uint32 intensity=4;
}

message PointCloud3 { 
    repeated PointI3 points=1;
}

service LidarService {
    rpc getScan(google.protobuf.Empty) returns (stream PointCloud3);
}